doc/source/release-notes/Percona-XtraDB-Cluster-5.6.34-26.19.rst

```diff
@@ -0,0 +1,72 @@
+.. rn:: 5.6.34-26.19
+
+===================================
+Percona XtraDB Cluster 5.6.34-26.19
+===================================
+
+Percona is glad to announce the release of
+|Percona XtraDB Cluster| 5.6.34-26.19 on December 9, 2016.
+Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/>`_
+or from our :ref:`software repositories <installation>`.
+
+Percona XtraDB Cluster 5.6.34-26.19 is now the current release,
+based on the following:
+
+* `Percona Server 5.6.34-79.1 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.34-79.1.html>`_
+
+* Galera Replication library 3.19
+
+* wsrep API version 26
+
+All Percona software is open-source and free.
+Details of this release can be found in the
+`5.6.34-26.19 milestone on Launchpad
+<https://launchpad.net/percona-xtradb-cluster/+milestone/5.6.34-26.19>`_.
+
+Fixed Bugs
+==========
+
+* PXC-252 / PXC-426 Optimized IST donor selection logic to avoid SST.
+  Child processes are now cleaned-up and node state is resumed if SST fails.
+
+* PXC-701 Added ``init.ok`` to the list of files that do not get removed during SST.
+
+* PXC-703 Fixed error with ASIO library when building Galera.
+
+* PXC-707 Fixed stalling of DML workload on slave node
+  caused by ``FLUSH TABLE`` executed on master.
+
+* PXC-713 Added ``encrypt=4`` mode for SST encryption
+  that uses SSL files generated by MySQL.
+  Modes ``1``, ``2``, and ``3`` will soon be deprecated.
+
+* PXC-715 Fixed ``super_read_only`` to not apply to Galera replication applier.
+
+* PXC-716 Redirected ``netcat`` output to ``stdout`` to avoid it in the log.
```

Comment:
Reference https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1625968 as it was publicly reported bug.